OracleTuxedo/core/samples/shm/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+docker build -t oracle/tuxedoshm .
+
+echo "To run the sample, use:"
+echo "docker run -ti -v \${Local_volumes_dir}/TuxedoVolumes/\${VERSION}:/u01/oracle/user_projects oracle/tuxedoshm /bin/bash"
```

Comment:
Don't suggest `-ti` and running `/bin/bash`. It should be run with `-d` in the background and a `CMD` or `ENTRYPOINT` script should handle whatever process needs to be run.